components/collapsible-panel/collapsible-panel.js

```diff
@@ -0,0 +1,405 @@
+import '../button/button.js';
+import '../colors/colors.js';
+import '../icons/icon.js';
+import '../icons/icon-custom.js';
+import '../expand-collapse/expand-collapse-content.js';
+import { css, html, LitElement } from 'lit';
+import { heading1Styles, heading2Styles, heading3Styles, heading4Styles } from '../typography/styles.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+const defaultHeading = 3;
+
+/**
+ * A container with a title that can be expanded/collapsed to show/hide content.
+ * @slot header - Slot for supporting header content
+ * @slot summary - Slot for the summary of the expanded content. Only accepts `d2l-collapsible-panel-summary-item`
+ * @slot default - Slot for the expanded content
+ * @slot actions - Slot for buttons and dropdown openers to be placed in top right corner of header
+ * @fires d2l-collapsible-panel-expand - Dispatched when the panel is expanded
+ * @fires d2l-collapsible-panel-collapse - Dispatched when the panel is collapsed
+ */
+class CollapsiblePanel extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			/**
+			 * REQUIRED: The title of the panel
+			 * @type {string}
+			 */
+			title: { type: String },
```

Comment:
`title` is actually a reserved HTML attribute -- it's how you get those ugly native tooltips to show up, and I'm actually seeing this show up as a tooltip whenever I hover anywhere on the panel. I don't think that's what you intended here? I'd recommend calling this something like `panel-title` instead.

Reply:
arg I always forget about the reserved attributes.....